src/model/test/SubSurface_GTest.cpp

```diff
@@ -1531,3 +1534,34 @@ TEST_F(ModelFixture, Issue_4361_Multi_Subsurfaces_Overlapping) {
   windowWallRatio = surface.windowToWallRatio();
   EXPECT_NEAR(windowWallRatio, 0.2742, 0.01);
 }
+
+TEST_F(ModelFixture, 4403_FenestrationAssembly) {
+  // Test for #4403 - Add Sql helper methods to retrieve U-factors, SHGC, or VT for glazing systems
```

Comment:
I like the test

(Side note: I'm not a big fan on relying on the ReverseTranslator though. Would be better with an OSM directly, but that's probably a non issue)

Reply:
it makes sense to use RUN_ENERGYPLUS to keep current, so I actually like this solution.

Reply:
I see pros/cons both ways. The con, as I see it, with OSM directly is that OSM/IDF get out of sync in `FrameAndDivider`.